hack/scripts/sync-prow.sh

```diff
@@ -36,4 +39,14 @@ git checkout sync
 # Import commits from old_prow/master into "sync" inside $new_prow.
 git remote add old_prow "$old_prow"
 git fetch old_prow
-git merge old_prow/master --no-edit
+# We expect this to fail with some merge conflicts due to documentation being moved to the site directory.
+git merge old_prow/master --no-edit --strategy-option theirs || true
+# Fix the merge conlflicts, these should be exclusively files that are considered [renamed/deleted] (i.e. moved).
+git add -u
+git commit -m "Sync Prow source from kubernetes/test-infra."
+
+# Rename go module paths
+find . -type f -exec sed -i 's/k8s.io\/test-infra/sigs.k8s.io\/prow/g' {} \;
```

Comment:
```suggestion
find . -type f -exec sed -i 's,k8s.io/test-infra,sigs.k8s.io/prow,g' {} \;
```

Reply:
Yeah that's more readable. I had just copied this from a prior PR's description. Updated.